config/release.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Can we do something that doesn't imply you should want to apply this to a k8s cluster, if we must have yaml? Would plain old 

```
image: github.com/knative/client/cmd/kn
```
work?

Reply:
I tried. It worked if we only keep "image: github.com/knative/client/cmd/kn"